spring-cloud-sleuth-zipkin-stream/src/main/java/org/springframework/cloud/sleuth/zipkin/stream/SamplingZipkinSpanIterator.java

```diff
@@ -38,6 +41,23 @@
  * @since 1.0.0
  */
 final class SamplingZipkinSpanIterator implements Iterator<zipkin.Span> {
+	private static final List<String> ZIPKIN_CLIENT_ANNOTATIONS = Arrays.asList(
+			Constants.CLIENT_ADDR, Constants.CLIENT_RECV, Constants.CLIENT_SEND,
+			Constants.CLIENT_RECV_FRAGMENT, Constants.CLIENT_SEND_FRAGMENT
+	);
+	private static final List<String> ZIPKIN_ANNOTATIONS = zipkinAnnotations();
+
+	private static List<String> zipkinAnnotations() {
```

Comment:
the most key annotations are CLIENT_SEND, SERVER_RECEIVE, CLIENT_ADDR, and SERVER_ADDR.  Since we control the instrumentation, we can focus on these.. ex we wouldn't send CLIENT_RECEIVE without CLIENT_SEND.

Bear in mind that Constants.LOCAL_COMPONENT is a tag/binaryAnnotation, so be careful with contains checks.